Libraries/Components/Slider/Slider.js

```diff
@@ -158,9 +184,10 @@ var Slider = React.createClass({
     props.onSlidingComplete = onSlidingComplete && ((event: Event) => {
       onSlidingComplete && onSlidingComplete(event.nativeEvent.value);
     });
-
-    return <RCTSlider 
+    thumbImage=resolveAssetSource(thumbImage);
```

Comment:
Nit: Don't reassign here just move this directly in the prop declaration (`thumbImage={resolveAssetSource(...)}`)